docs/orcfax-feeds/feed-overview.md

```diff
@@ -53,77 +35,144 @@ For a feed of type CER, the naming convention is
 [cex-3]: https://tradenation.com/articles/base-currency-and-quote-currency/
 
-#### CEX showcase
+## Subsidized feeds
 
-In order to demonstrate the kinds of feeds derived from CEXes which integrators
-could utilize in their dApps, Orcfax is currently offering the following feeds
-as a showcase:
+There are feeds which Orcfax has identified for their broad utility across
+Cardano projects and has therefore committed to supporting their publication in
+order to aid in development initiatives. These feeds have been selected in an
+effort to provide developers with opportunities to design their solutions from
+the beginning with authentic and verifiable data as a foundation.
 
-| Feed name         | Start Date                 | End date\* |
-| ----------------- | -------------------------- | ---------- |
-| [ADA-BTC][adabtc] | Sep 12th 2024 17:01:24 UTC | TBA        |
-| [BTC-USD][btcusd] | Sep 12th 2024 17:01:24 UTC | TBA        |
+A subsidized feed is one which Orcfax has committed to maintaining even without
+a sponsor.
 
-\*As these are showcase feeds, Orcfax makes no guarantees as to how long they
-will be offered in this capacity; integrators who would like to take advantage
-of these feeds or are interested in learning more about our services, please
-contact us via [email][email] or [Discord][discord].
+Orcfax is currently offering the following CEX subsidized feeds:
 
-[email]: mailto:info@orcfax.io
-[discord]: https://discord.com/invite/UbAeRuNzDu
+| Feed name | Start Date                 | End date |
+| --------- | -------------------------- | -------- |
+| ADA-USD   | Sep 12th 2024 17:01:24 UTC | TBA      |
 
-### Feeds derived from DEXes
+Orcfax is currently offering the following DEX subsidized feeds:
 
-The price values published through these feeds are derived entirely from
-decentralized exchange ([DEX][dex-1]) data.
+| Feed name | Start Date                 | End date |
+| --------- | -------------------------- | -------- |
+| FACT-ADA  | Sep 12th 2024 17:01:24 UTC | TBA      |
 
-For these feeds, ordering defers to how pairs appear on DEXes.
+when an integrator activates a subscription for a subsidized feed it will no
+longer be identified as such and will instead be labeled a
+[sponsored][sponsored] feed.
 
-[dex-1]: https://glossary.orcfax.io/#dex
+[sponsored]: #sponsored-feeds
+
+### Sunsetting subsidized feeds
+
+In contrast to [showcase][showcase] feeds, subsidized feeds can be used in
+production because Orcfax has agreed to cover the cost of their support.
+
+While Orcfax has committed to maintaining access to these feeds, integrators who
+wish to utilize these feeds without a subscription should do so at their own
+risk; Orcfax still reserves the right to adjust or remove support for these
+feeds as business needs dictate. However, any change to service relating to
+subsidized feeds will be announced three months in advance in order to provide
+integrators with adequate forewarning.
```

Comment:
This part:

> any change to service relating to
subsidized feeds will be announced three months in advance in order to provide
integrators with adequate forewarning.

Ideally will be a the top of this section using plain-language.

E.g.

> Changes to service relating to subsidized feeds will be announced three months in advance in order to provide integrators with adequate forewarning.

Alternatively (my preference), the language more terse:

> Orcfax has agreed to cover the cost of their support of subsidized feeds enabling them to be used in production. Changes to services relating to subsidized feeds will be announced __three months__ in advance to provide integrators with adequate forewarning.
```